raincloud/models/service.py

```diff
@@ -0,0 +1,15 @@
+import os
+import json
+
+class Service(object):
```

Comment:
Although we are not using a DB, it's still useful to take create objects to better stick to OOP